app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -191,7 +191,8 @@ def parse_openshift_image(openshift_image)
             :author         => docker_metadata[:Author],
             :docker_version => docker_metadata[:DockerVersion],
             :size           => docker_metadata[:Size],
-            :labels         => parse_labels(openshift_image)
+            :labels         => parse_labels(openshift_image),
+            :registered_on  => docker_metadata[:Created] ? Time.parse(docker_metadata[:Created]).utc : nil
```

Comment:
Do we know how `docker_metadata[:Created]` relates to `openshift_image[:metadata][:creation_timestamp]`?
(Wild guess: when you import an year-old image from docker.io into openshift's registry, I suspect former will be a year ago — when the image itself was built — and latter will be now.)